src/smtsolvers/CoreSMTSolver.h

```diff
@@ -417,6 +417,8 @@ class CoreSMTSolver
 
     void     relocAll         (ClauseAllocator& to);
 
+    virtual void runPeriodic () { return; } ;            // Check if solver is in clause share mode and leanr clauses
```

Comment:
The comment seems to be outdated.  And there's no need for the semicolons here and on line 418, wherever they came from :)

Reply:
Done